src/main/java/com/brainpix/profile/controller/PortfolioController.java

```diff
@@ -0,0 +1,85 @@
+package com.brainpix.profile.controller;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.brainpix.CommonPageResponse;
+import com.brainpix.api.ApiResponse;
+import com.brainpix.profile.dto.request.PortfolioRequest;
+import com.brainpix.profile.dto.response.PortfolioDetailResponse;
+import com.brainpix.profile.dto.response.PortfolioResponse;
+import com.brainpix.profile.service.PortfolioService;
+
+import lombok.RequiredArgsConstructor;
+
+@RestController
+@RequestMapping("/portfolios")
+@RequiredArgsConstructor
+public class PortfolioController {
+
+	private final PortfolioService portfolioService;
+
+	@PostMapping
+	public ResponseEntity<ApiResponse<Long>> createPortfolio(
```

Comment:
long타입으로 보내버리면 프런트분들이 어떤 역할의 데이터인지 모르실 수 있어서 dto로 감싸는게 좋을거 같아요
```suggestion
	public ResponseEntity<ApiResponse<CreatePortfolioDto.Response>> createPortfolio(
    
    public class CreatePortfolioDto {
        public static class Response {
            private Long portfolioId;
        }
    }
```
이렇게 하면 data라는 응답값 대신 `"data" : { "portfolioId" : 3 } ` 이런 식으로 갈거같습니다